src/zenml/metadata_stores/mysql_metadata_store.py

```diff
@@ -26,10 +26,10 @@
 class MySQLMetadataStore(BaseMetadataStore):
     """MySQL backend for ZenML metadata store."""
 
-    username: Optional[str] = None
+    port: int = 3306
+    username: Optional[str] = "root"
     password: Optional[str] = None
```

Comment:
```suggestion
    password: Optional[str] = ""
```

Reply:
In this case might make sense to have a default database name as well?